moleculegen/tests/test_vocab.py

```diff
@@ -52,9 +58,12 @@ def test_corpus(self):
             # for data sampling and model fitting.
             tokens = Token.augment(tokens)
             # Test id-to-token mapping.
-            self.assertListEqual(self.vocab.get_tokens(idx), list(tokens))
+            self.assertEqual(
+                ''.join(self.vocab.get_tokens(idx)),
+                tokens,
+            )
             # Test token-to-id mapping.
-            self.assertListEqual(idx, self.vocab[tokens])
+            self.assertListEqual(idx, self.vocab[Token.tokenize(tokens)])
```

Comment:
To get a list of indexes, we should first tokenize our SMILES string `tokens` and then call `Vocabulary.__getitem__`. Previously, the tokenization was not necessary since it was trivial (list all symbols as single-char tokens). Now, we use an improved tokenization.